cmd/renterd/web.go

```diff
@@ -133,20 +133,22 @@ func (t treeMux) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 func startWeb(l net.Listener, node *node, password string) error {
 	renter := api.NewServer(&syncer{node.g, node.tp}, &chainManager{node.cm}, txpool{node.tp}, node.w, node.hdb, rhpImpl{}, node.cs, newSlabMover(), node.os)
+	auth := jape.BasicAuth(password)
```

Comment:
`nit` separate PR

Reply:
should have been directly pushed to master, you mean :P